src/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -159,17 +160,8 @@ private static function getGapicVersion()
      *
      *     @type string $serviceAddress Required. The domain name of the API remote host.
      *     @type mixed $port The port on which to connect to the remote host. Default 443.
-     *     @type \Grpc\Channel $channel
```

Comment:
Do we want to remove all these args? Or will we still support them, but document them as only being available with grpc transport?

Reply:
I was envisioning documenting them in the GrpcTransport, but we can leave them documented here as well.